internal/oci/runtime_oci.go

```diff
@@ -630,11 +631,38 @@ func (r *runtimeOCI) DeleteContainer(c *Container) error {
 	return err
 }
 
+func updateContainerStatusFromExitFile(c *Container) error {
+	exitFilePath := filepath.Join(c.dir, "exit")
+	fi, err := os.Stat(exitFilePath)
+	if err != nil {
+		return fmt.Errorf("failed to find container exit file for %v: %v", c.id, err)
+	}
+	c.state.Finished, err = getFinishedTime(fi)
+	if err != nil {
+		return fmt.Errorf("failed to get finished time: %v", err)
+	}
+	statusCodeStr, err := ioutil.ReadFile(exitFilePath)
+	if err != nil {
+		return fmt.Errorf("failed to read exit file: %v", err)
+	}
+	statusCode, err := strconv.Atoi(string(statusCodeStr))
+	if err != nil {
+		return fmt.Errorf("status code conversion failed: %v", err)
+	}
+	c.state.ExitCode = utils.Int32Ptr(int32(statusCode))
+	return nil
+}
+
```

Comment:
Let's wrap the errors here please :)

Reply:
do you mean use errors.Wrapf instead of fmt.Errorf("%v")?